app/javascript/check_ins/store.ts

```diff
@@ -9,9 +10,11 @@ import { Bootstrap, NeedSatisfactionRating } from './types';
 
 declare const Routes: typeof RoutesType;
 
+const bootstrap = untypedBootstrap as Bootstrap;
+
 export const useCheckInsStore = defineStore('check-ins', {
   state: () => ({
-    ...(window.davidrunger.bootstrap as Bootstrap),
```

Comment:
For the sake of DRY and also (at least in some cases) better type safety, I removed all remaining direct references to `window.davidrunger.bootstrap`. Now, we only access bootstrap data via `@/lib/bootstrap`.